packages/contracts/deploy/helpers.ts

```diff
@@ -1,6 +1,14 @@
 import {promises as fs} from 'fs';
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
 
+import {ensLabelHash, ensDomainHash} from '../utils/ensHelpers';
+
+// TODO: Add support for L2 such as Arbitrum. (https://discuss.ens.domains/t/register-using-layer-2/688)
+export const ENS_ADDRESSES: {[key: string]: string} = {
+  mainnet: '0x314159265dD8dbb310642f98f50C066173C1259b',
+  ropsten: '0x112234455C3a32FD11230C42E7Bccd4A84e02010',
```

Comment:
Are these addresses the correct ones? AFAIK the official ENS registry starts with many zeroes